File: src/Test/TCOFFICheatcode.go
package Test_TCOFFICheatcode


func RunTCOFFICheatcode(limit float64) float64 {
	n := int(limit)
	acc := 0
	for i := n; i > 0; i-- {
		acc += 1
	}
	return float64(acc)
}

File: src/Test/TCOFFICheatcode.cc
#include "purescript.h"
FOREIGN_BEGIN( Test_TCOFFICheatcode )
exports["runTCOFFICheatcode"] = [](const boxed& in) -> boxed {
    int acc = 0;
    for (int n = unbox<int>(in); n > 0; --n) acc += n % 3;
    return acc;
};
FOREIGN_END
